src/_ert/forward_model_runner/reporting/message.py

```diff
@@ -35,7 +35,7 @@ class ChecksumDict(_ChecksumDictBase, total=False):
 
 @dataclasses.dataclass
 class MemoryStatus:
-    """Holds memory information that can be represented as a line of CSV data"""
+    """Holds processtree information that can be represented as a line of CSV data"""
```

Comment:
We could consider renaming this class slightly since it now contains more data than just memory related stuff.

Reply:
Yes, `ProcessTreeStatus` might cut it.